src/utils/getMaxNumberId.tsx

```diff
@@ -0,0 +1,11 @@
+import { Todo } from '../types';
+
+export const getMaxNumberId = (todos: Todo[]): number => {
+  if (todos.length === 0) {
```

Comment:
it will be more readable. Use this syntax in the future
```suggestion
  if (!todos.length) {
```